packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -1,15 +1,17 @@
 import { assert, details as X, q, Fail } from '@agoric/assert';
+import {
+  zeroPad,
+  makeEncodePassable,
+  makeDecodePassable,
+  isEncodedRemotable,
+} from '@endo/marshal/src/encodePassable.js';
```

Comment:
I think we're shooting for a stricter quality standard in Endo. Does that include forbidding deep imports?

OIC, https://github.com/Agoric/agoric-sdk/pull/6792/files#diff-4271898ca5dee9abbed8453730a6cea4c55a3630f9addad6cb4b444aeaf5d454

Reply:
Yup, https://github.com/Agoric/agoric-sdk/pull/6792 is coming once we can depend on the endo in which all these are exported. That's why #6792 currently fails in CI.